doc/src/fpga_api/quick_start/readme.md

```diff
@@ -393,6 +393,79 @@ Done
 
 ```
 
+## Setup IOFS Release1 Bitstream on FPGA PCIe card  ##
+
+Program IOFS Release1 bitstream on FPGA D5005 or N6010 cards and reboot system.
+
+Run command: lspci | grep acc
+
+```3b:00.0 Processing accelerators: Intel Corporation Device af00 (rev 01)```
+
+Number of virtual functions supported by bitstream
+
+``` 
+   cat /sys/bus/pci/devices/0000\:3b\:00.0/sriov_numvfs 
+   output: 3
+```
+
+Enable FPGA virtual functions
+
+``` 
+   sudo sh -c "echo 3 > /sys/bus/pci/devices/0000\:3b\:00.0/sriov_numvfs"
+```
+
+List of FPGA PF and VF's
+
+``` 
+3b:00.0 Processing accelerators: Intel Corporation Device af00 (rev 01)
+3b:00.1 Processing accelerators: Intel Corporation Device af01 (rev 01)
+3b:00.2 Processing accelerators: Intel Corporation Device af01 (rev 01)
+3b:00.3 Processing accelerators: Intel Corporation Device af01 (rev 01)
+``` 
+
+FPGA VF1/3b:00.1/Host exerciser loopback Accelerator guid 56E203E9-864F-49A7-B94B-12284C31E02B
```

Comment:
Will they always appear in this order?  If not, we should give instructions on how to determine the AFU ID from sysfs.

Reply:
updated doc with fpgainfo port, it lists AFU id